src/components/Form/FileUpload/FileUpload.test.tsx

```diff
@@ -286,6 +287,45 @@ describe("upload action", () => {
     expect(onChange).toHaveBeenCalledTimes(1);
   });
 
+  it("shows error when only single file upload enabled", async () => {
```

Comment:
Each test should start with `should` keyword:
```suggestion
  it("should show error when trying to upload multiple files but only single file upload is enabled", async () => {
```